api/jobs_test.go

```diff
@@ -61,14 +61,20 @@ func expectedJob(id string,
 	searchIndexName string,
 	state string,
 	totalSearchDocuments int,
-	totalInsertedSearchDocuments int) models.Job {
+	totalInsertedSearchDocuments int,
+	version string) (models.Job, error) {
+
+	cfg, err := config.Get()
```

Comment:
**COMMENT 2**
Whenever `expectedJob` is called in the test, the `config.Get()` is called beforehand within the test. You don't need to call this again to get the BindAddr. 

You just need to add a new parameter `bindAddr` in the function and then use that - and then pass `cfg.BindAddr` which is available to use in the test